### DISCUSSION TITLE
Suggestions for making Frigate (and included Go2rtc) more resistant for network instability

### DISCUSSION THREAD

**Discussion by quenthal** [blakeblackshear/frigate#5418] [Ask A Question]
Last night my UDMPRO autoupdated UniFi OS and Network-capabilities. Almost at the same time Frigate lost connection to  2 out of my 7 cameras. Only remedy was to restart frigate container.

Any suggestions for avoiding this or making it more resistant or "self-heal" for this? Could we make it so that docker container would exit on this kind of errors? Then I could start the container with -restart? Or some more elegant way to manage this, since this suggestion would probably make some other case worse...

**Comment by NickM-27** [comment]
Frigate is designed to reconnect gracefully. There's a known issue in the current beta which means when this happens go2rtc / frigate can't negotiate a good connection. It's already been fixed for the next beta.
